DSharpPlus/Entities/Interaction/DiscordFollowupMessageBuilder.cs

```diff
@@ -181,9 +193,10 @@ public DiscordFollowupMessageBuilder AddFile(FileStream stream, bool resetStream
         /// </summary>
         /// <param name="files">Dictionary of file name and file data.</param>
         /// <param name="resetStreamPosition">Tells the API Client to reset the stream position to what it was after the file is sent.</param>
+        /// <returns>The builder to chain calls with.</returns>
         public DiscordFollowupMessageBuilder AddFiles(Dictionary<string, Stream> files, bool resetStreamPosition = false)
         {
-            if (this.Files.Count() + files.Count() >= 10)
+            if (this.Files.Count + files.Count >= 10)
```

Comment:
There is a bug here, although you didn't introduce it: It should be > 10 instead of >= 10, because this is already taking both the amount of existing files and added files into account, unlike the other overloads, therefore this will never allow you to add the 10th file with this method.